pkg/ipamd/ipamd.go

```diff
@@ -440,7 +438,7 @@ func (c *IPAMContext) updateCIDRsRulesOnChange(oldVPCCidrs []string) []string {
 	var pbVPCCIDRs []string
 	newVPCCIDRs := c.awsClient.GetVPCIPv4CIDRs()
 	for _, cidr := range newVPCCIDRs {
```

Comment:
I think this `for` loop can be removed, remove `pbVPCCIDRs`, and just use `newVPCCIDRs` throughout the rest of this function.